moveit_ros/planning/planning_pipeline/src/planning_pipeline.cpp

```diff
@@ -319,7 +319,11 @@ bool planning_pipeline::PlanningPipeline::generatePlan(const planning_scene::Pla
       arr.markers.push_back(m);
 
       std::vector<std::size_t> index;
-      if (!planning_scene->isPathValid(*res.trajectory, req.path_constraints, req.group_name, false, &index))
+      if(!res.trajectory->isContinuous(0.1)){
```

Comment:
shouldn't this be configurable somehow? Or depend on some existing parameters?

Reply:
@simonschmeisser I decided to remove the default check because the issue we originally seem to be caused by the code hierarchically above the planning pipeline. So this PR just adds some nice to have functionality.